internal/k8s/synclet.go

```diff
@@ -0,0 +1,35 @@
+package k8s
+
+import (
+	"github.com/docker/distribution/reference"
+	"github.com/windmilleng/tilt/internal/synclet/sidecar"
+)
+
+func InjectSyncletSidecar(entity K8sEntity, matchRef reference.Named) (K8sEntity, bool, error) {
```

Comment:
FFTI, but I'd have a slight preference for putting this in the sidecar package, since it feels kinda weird for k8s to know about synclets, and the sidecar package already has a k8s dep anyway (and is kind of explicitly about mucking with k8s?). (Or, if we're keeping this here, it still seems like maybe the synclet entity defs should be next to this function?)

Reply:
done!